Tensile/Source/lib/include/Tensile/ContractionSolution.hpp

```diff
@@ -144,6 +181,22 @@ namespace Tensile
                                                       double totalGranularity,
                                                       int    globalSplitU) const;
 
+        TAMetricProblemScore computeProblemScore(Hardware const& hardware,
+                                                 double          M,
+                                                 double          N,
```

Comment:
Why is there a disparity between using size_t and double types for M, N and K values? These are integral values, no?